perf_dashboard/deploy/perf-dashboard/values.yaml

```diff
@@ -0,0 +1,14 @@
+maxReplicas: 5
+replicas: 3
+minReplicas: 1
+CPUUtilization: 80
+
+fullnameOverride: perf-dashboard
+ipName: perf-dashboard
+domain: perf.dashboard.istio.io
+
+image: gcr.io/istio-testing/perf-dashboard
+version: latest
```

Comment:
so this `latest` version is what you mentioned the timestamp version?

Reply:
I put that there as a _default_. When this is deployed the version is computed from the [date and sha](https://github.com/istio/tools/blob/73bf3fe96d70e0a6497065147ec26ce841b717d0/perf_dashboard/Makefile#L1). (e.g. `v20200321-67dfdb7`)